tests/accordion-test.ts

```diff
@@ -0,0 +1,424 @@
+
+        const activeItems = el.querySelectorAll(".acc-active");
+        activeItems.forEach((item) => {
+            expect(
+                getComputedStyle(item.querySelector(".acc-body")).display
+            ).to.equal("block");
+        });
+
+        const inactiveItems = el.querySelectorAll(
+            ".container:not(.acc-active) .acc-body"
+        );
+        inactiveItems.forEach((item) => {
+            expect(getComputedStyle(item).display).to.equal("none");
+        });
+    });
+
+    it("allows multiple items to be open if nomultiple is true then set as false", async () => {
+        const el = await litFixture(
+            createAccordionTemplate(
+                { activecls: "acc-active", nomultiple: true },
+                [
+                    {
+                        title: "What is Artificial Intelligence?",
+                        body: "body 1",
+                        active: true,
+                    },
+                    {
+                        title: "What is Machine Learning?",
+                        body: "body 2",
+                        active: false,
+                    },
+                    { title: "title 3?", body: "body 3", active: true },
+                    { title: "title 4?", body: "body 4", active: true },
+                    { title: "title 5?", body: "body 5", active: false },
+                ]
+            )
+        );
+        const titles = el.querySelectorAll(".acc-title");
+
+        expect(el.hasAttribute("nomultiple")).to.be.true;
+        expect(el.querySelectorAll(".acc-active").length).to.equal(1);
+        const active = el.querySelectorAll(".acc-active");
+        active.forEach((item) => {
+            expect(
+                getComputedStyle(item.querySelector(".acc-body")).display
+            ).to.equal("block");
+        });
+        titles[0].click();
+        titles[1].click();
+        titles[2].click();
+        await el.updateComplete;
+
+        expect(el.querySelectorAll(".acc-active").length).equal(1);
+
+        expect(titles[2].parentElement?.classList.contains("acc-active"));
+        expect(getComputedStyle(titles[2]).display).to.equal("block");
+        el.removeAttribute("nomultiple");
+        await el.updateComplete;
+
+        titles[2].click();
+        await el.updateComplete;
+        expect(el.querySelectorAll(".acc-active").length).equal(0);
+        expect(el.hasAttribute("nomultiple")).to.be.false;
+        titles[0].click();
+        expect(titles[0].parentElement?.classList.contains("acc-active")).to.be
+            .true;
+        expect(titles[2].parentElement?.classList.contains("acc-active")).to.be
+            .false;
+        expect(titles[1].parentElement?.classList.contains("acc-active")).to.be
+            .false;
+        expect(titles[3].parentElement?.classList.contains("acc-active")).to.be
+            .false;
+        expect(titles[4].parentElement?.classList.contains("acc-active")).to.be
```

Comment:
Better to open multiple items and check if man accordion items can be open at the same time.